ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.20__periodics.yaml

```diff
@@ -169,6 +183,26 @@ tests:
       TEST_SUITE: openshift/machine-config-operator/disruptive
     workflow: baremetalds-e2e-ovn-dualstack
   timeout: 6h0m0s
+- as: e2e-aws-ovn-ocl
+  cron: 0 8 * * 1
+  steps:
+    cluster_profile: aws-3
+    env:
+      MCO_CONF_DAY2_OCL_POOLS: master worker
+    test:
+    - chain: mco-test-e2e-ocl-aws
+    workflow: cucushift-installer-rehearse-aws-ipi
+  timeout: 6h0m0s
+- as: e2e-aws-ovn-upgrade-ocl
+  cron: 0 8 * * 1
+  steps:
+    cluster_profile: aws-3
+    env:
+      MCO_CONF_DAY2_OCL_POOLS: master worker
+    test:
+    - chain: mco-test-e2e-upgrade-ocl-aws
+    workflow: cucushift-installer-rehearse-aws-ipi
```

Comment:
Hmm, similar question, it seems that `cucushift` is used mostly for QE related tests, is there a reason we're using them for non-QE periodics?